src/components/App.js.jsx

```diff
@@ -3,8 +3,23 @@ import { Link } from 'react-router'
 import '../css/App.css'
 import BuilderContainer from './containers/BuilderContainer.js.jsx'
 import DashboardContainer from './containers/DashboardContainer.js.jsx'
+import Ajax from './helpers/ajax.js'
+import { connect } from 'react-redux' 
 
 class App extends Component {
+
+  componentDidMount() {
+    const api = new Ajax()
+    api.fetchTags().then(response => this.storeTags(response.data))
```

Comment:
@wfro so `fetchTags()` would just be a function that returns a `{ action: 'STORE_TAGS', payload: { tags: tags } }` object?